webapp/channels/src/components/user_settings/sidebar/show_unreads_category/show_unreads_category.tsx

```diff
@@ -75,6 +80,11 @@ export default class ShowUnreadsCategory extends React.PureComponent<Props, Stat
     };
 
     handleSubmit = async () => {
+        if (!this.props.currentUserId) {
+            // Only for type safety, won't actually happen
```

Comment:
Should these props be changed to be required? I would expect `currentUserId` to always be defined, and I think it would make sense for `adminMode` to be required as well

Reply:
Done.